tests/socketcall_32bit/Makefile

```diff
@@ -0,0 +1,11 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+
+conn: conn.c
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
I have a number of binaries compiled now, including a 32-bit static version.  It is mildly tempting to check in this binary as part of the test so that we don't need to have the 32-bit build or runtime libraries installed on all target test machines, but still have a test suite build target available to rebuild the binary from source should the need arise.  As expected, the the static binary is 3/4MB while the dynamic one is 10KB so it is a bit hard to swallow.